clients/banking/src/components/AccountArea.tsx

```diff
@@ -159,6 +159,13 @@ export const AccountArea = ({ accountMembershipId }: Props) => {
     setIsScrolled(event.nativeEvent.contentOffset.y > 0);
   }, []);
 
+  useEffect(() => {
+    const intervalId = setInterval(() => {
+      void fetch("/api/ping", { method: "POST" });
+    }, 30000);
```

Comment:
I think it will be better to put this in a variable with as comment the value in seconds